packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -691,4 +691,53 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('With timezone', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone'));
+
+    Terra.it.matchesScreenshot({ selector: '#root' });
+  });
+
+  describe('With timezone', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone-missing-hour');
+    });
+
+    it('missing hour', () => {
```

Comment:
This test step uses a test that has the timezone set so it is more appropriate. We can remove the [test step done here](https://github.com/cerner/terra-framework/blob/ca8fde828d309978e69ab801d59044f5e71626b0/packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js#L70-L82) since it is a duplicate and was using the local timezone, which may not always honor daylight savings. 

Additionally, [this test step](https://github.com/cerner/terra-framework/blob/ca8fde828d309978e69ab801d59044f5e71626b0/packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js#L84-L94) should run against the `date-time-picker-with-timezone-missing-hour` instead of the `date-time-picker-default` to stay consistent.

Reply:
changed here 15688a34fa0eff270ee5c526ff6e65624b2d5fa1